src/autocomplete/mysql/grammar/MySqlParser.g4

```diff
@@ -602,10 +602,10 @@ alterSpecification
     | DROP COLUMN? uid RESTRICT?                                                                                                      # alterByDropColumn
     | DROP (CONSTRAINT | CHECK) uid                                                                                                   # alterByDropConstraintCheck
     | DROP PRIMARY KEY                                                                                                                # alterByDropPrimaryKey
-    | DROP indexFormat = (INDEX | KEY) uid                                                                                            # alterByDropIndex
-    | RENAME indexFormat = (INDEX | KEY) uid TO uid                                                                                   # alterByRenameIndex
+    | DROP indexFormat = (INDEX | KEY) indexName                                                                                            # alterByDropIndex
+    | RENAME indexFormat = (INDEX | KEY) indexName TO uid                                                                                   # alterByRenameIndex
     | ALTER COLUMN? uid ( SET DEFAULT ( stringLiteral | LR_BRACKET expression RR_BRACKET) | SET (VISIBLE | INVISIBLE) | DROP DEFAULT) # alterByAlterColumnDefault
-    | ALTER INDEX uid (VISIBLE | INVISIBLE)                                                                                           # alterByAlterIndexVisibility
+    | ALTER INDEX indexName (VISIBLE | INVISIBLE)                                                                                           # alterByAlterIndexVisibility
```

Comment:
The idea is to have a rule for reading the existent index name, and to catch it with the `preferredRules`